pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -72,22 +94,48 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Replica Quiescence" sources={storeSources}>
+    <LineGraph
+      title="Replica Quiescence"
+      sources={storeSources}
+      tooltip={
+        `The total number of replicas and quiesced replicas.
+        Quiesced replicas are ones that have not been accessed in a while.`
```

Comment:
`Replicas are considered quiesced when they haven't been accessed in <PRECISE TIME FRAME>`, perhaps.

Also suggest putting a `<br />` above to force a line break.